### ISSUE TITLE
Paper and checkpoint

### ISSUE THREAD

**Issue by Darius-H** [wsntxxn/DCASE2022T6_CLAP#1]
Is this repository the code implementation of https://dcase.community/documents/challenge2022/technical_reports/DCASE2022_Xu_106_t6a.pdf ?
Is the pretrained checkpoint available?

**Comment by wsntxxn** [comment]
Yes, it is the implementation of this technical report. We provide a [checkpoint](https://github.com/wsntxxn/DCASE2022T6_CLAP/releases/tag/v1.0.0).
